src/tests/ftest/pool/attribute.py

```diff
@@ -67,11 +68,13 @@ def verify_list_attr(self, indata, size, buff):
 
         """
         # length of all the attribute names
-        aggregate_len = sum(
-            len(attr_name) if attr_name is not None else 0
-            for attr_name in list(indata.keys())) + 1
+        aggregate_len = 0
+        for attr_name in indata.values():
```

Comment:
This gets the attribute values (values of the dictionary) not the attribute names (dictionary keys):
```suggestion
        for attr_name in indata.keys():
```

Reply:
It is true, it needs to only use values()